docs/SNIPPET.txt

```diff
@@ -1 +1,2 @@
-TODO: add snippet
+func hello(name):
+  return "Hello, %s!" % name
```

Comment:
```suggestion
	return "Hello, %s!" % name
```
A nice and simple example, I like it :) It also shows how little boilerplate code is required in GDScript :P

I'd like to suggest only one change: by default Godot's editor uses tabs for indenting the code. I don't think it's required (I'm pretty sure that spaces will work as well, just like in Python) but the default experience that users will get when using Godot Engine is tabs. Could you please update the example accordingly?